src/pages/NewInvoicePage/index.jsx

```diff
@@ -0,0 +1,94 @@
+import { useState } from "react";
+import InvoiceItem from "../../components/InvoiceItem";
+import NewInvoiceDetails from "../../components/NewInvoiceDetails";
+import { Link } from "react-router-dom";
+
+export default function NewInvoicePage() {
+
+    const [list, setList] = useState([{ quantity: 0, rate: 0, total: 0, description: '' }]);
+
+    const addItem = () => {
+        setList([...list, { quantity: 0, rate: 0, total: 0, description: '' }]);
+    };
+
+    const [clientId, setClientId] = useState('')
+
+    const removeItem = () => {
+        if (list.length > 1) {
+            setList(list.splice(0, list.length - 1))
+        }
+    }
+
+    let megaTotal = list.reduce((carry, item) => carry + item.total, 0)
+
+    
+
+    function updateQuantity(index, quantitySum) {
+        const listCopy = [...list]
+        listCopy[index].quantity = quantitySum
+        setList(listCopy)
+    }
+    function updateRate(index, rateSum) {
+        const listCopy = [...list]
+        listCopy[index].rate = rateSum
+        setList(listCopy)
+    }
+    function updateDesc(index, description) {
+        const listCopy = [...list]
+        listCopy[index].description = description
+        setList(listCopy)
+    }
+    function updateTotal(index, total) {
+        const listCopy = [...list]
+        listCopy[index].total = total
+        setList(listCopy)
+    }
+
+    function updateClient(clientIdNew) {
+        setClientId(clientIdNew)
+    }
+
+    function createInvoice() {
+        let newInvoice = {
+            client: clientId,
+            total: megaTotal,
+            details: list
+        }
+        console.log(newInvoice)
+        fetch('https://invoicing-api.dev.io-academy.uk/invoices', {
+            method: "POST",
+            body: JSON.stringify(newInvoice),
+            headers: {
+                "Content-Type": "application/json",
+                "Accept": "application/json"
+            }
+        }).then(res => res.json()).then(data => {
+            console.log(data)
```

Comment:
Pull out this log as well.

Reply:
We need to do something when the invoice gets added. I think it would be good if you displayed a success message under the bottons, along with a link to go back home